example/lib/payment_slection_page.dart

```diff
@@ -65,8 +68,9 @@ class _PaymentSelectionPageState extends State<PaymentSelectionPage> {
   }
 
   fetchAllPaymentMethods() {
-    _razorpay.getPaymentMethods().then((value) {
+    _razorpay.getPaymentMethods({}).then((value) {
```

Comment:
@ramprasadAnand is this required here? If it is empty, can we add it ourselves? Also, did you test this using the await function as well?

Reply:
Just for testing propose i have added this as an empty object, But user can send any payload here. But payload is required. async / await testing and working.

Reply:
The object is required in the case of subscription ID right? Can we have an overloading function that can do this, rather than the user having to send empty objects for the majority of the cases? @ramprasadAnand

Reply:
Handled this scenario by making the parameter optional. @vivekshindhe